### PR TITLE
rendering all modules and some styling

### PR DESCRIPTION
**Pull request by azizbouland**
@twzkraus Thanks!

### CODE REVIEW

**Comment by azizbouland** [comment]
Umm seems like there's something wrong, I have +1 million files added, I'll look into that..

**Comment by twzkraus** [comment]
Lol oops. Looks like you need to ignore node_modules/. Feel free to tag me again when it's fixed

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Product Options</title>
```
Comment (twzkraus) [inline public/index.html]: Probably change this to Target or something

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="appWrapper">
+      <div class="storePath">Target / Men / Men's Clothing / Hoodies & Sweatshirts</div>
+      <div class="itemNameHeader">Men's Standard Fit French Terry Hoodie Sweatshirt - Goodfellow & Co™</div>
+      <div class="ShopAll">Shop all Goodfellow & Co</div>
```
Comment (twzkraus) [inline public/index.html:11]: are these 3 div's necessary? Seems they might be carry-overs from productOptions

**Review by twzkraus — COMMENTED** [review]
A couple minor comments. Pretty sure I reviewed all the files that were relevant here.

File: public/index.html (hunk 2 above)
Comment (azizbouland) [reply public/index.html:11]: Those add the title and other stuff at the top
Screen Shot 2020-10-07 at 9 39 00 PM

Comment (twzkraus) [reply public/index.html:11]: Got it. Makes sense.

**Pull request merged**